src/log.js

```diff
@@ -1,4 +1,5 @@
+let adb = require('./adb');
+
 module.exports = function log(req, res) {
-  // TODO
-  res.status(501).send('501 Not Implemented');
+  adb.logcat().pipe(res);
```

Comment:
Will ending the HTTP stream propagate the end back to the stream? I guess what I'm asking is: if the request ends, will the stream end and the process stopped? We manually manage that interaction in Raptor.